schema_based_on_diagram.sql

```diff
@@ -8,5 +8,28 @@ CREATE TABLE medical_histories (
     id SERIAL PRIMARY KEY,
     admitted_at TIMESTAMP,
     FOREIGN KEY patient_id INT REFERENCES patients(id),
-    status VARCHAR(80),
-);
\ No newline at end of file
+    status VARCHAR(80)
+);
+
+CREATE TABLE invoices (
+    id SERIAL PRIMARY KEY,
+    total_amount DECIMAL(10,2) NOT NULL,
+    generated_at TIMESTAMP,
+    payed_at TIMESTAMP,
+    FOREIGN KEY medical_history_id INT REFERENCES medical_histories(id)
+);
+
+CREATE TABLE treatments (
+    FOREIGN KEY id INT REFERENCES medical_histories(id),
+    type VARCHAR(50) NOT NULL,
+    name VARCHAR(50) NOT NULL
+);
```

Comment:
Great job team 🤝

- [ ] As stated in the project requirements, kindly ensure all tables created have primary keys.